tests/factories.py

```diff
@@ -46,7 +46,8 @@
 from website.addons.wiki.model import NodeWikiPage
 from tests.base import fake
 
-from tests.base import DEFAULT_METASCHEMA
+from tests.base import get_default_metaschema
+DEFAULT_METASCHEMA = get_default_metaschema()
```

Comment:
This method is called at import time, possibly before the test database is set up. As discussed in person, that may not be the intended result.